index.html

```diff
@@ -1,68 +1,95 @@
-  Engage your clients through compelling visuals that do most of the marketing for you.
-
-  Learn more
-
-  Stand out to the right audience
-
-  Using a collaborative formula of designers, researchers, photographers, videographers, and copywriters, we’ll build and extend your brand in digital places. 
-
-  Learn more
-
-  Graphic design
-  Great design makes you memorable. We deliver artwork that underscores your brand message and captures potential clients’ attention.
-
-  Photography
-  Increase your credibility by getting the most stunning, high-quality photos that improve your business image.
-
-  Client testimonials
-
-  We put our trust in Sunnyside and they delivered, making sure our needs were met and deadlines were always hit.
-  Emily R.
-  Marketing Director
-
-  Sunnyside’s enthusiasm coupled with their keen interest in our brand’s success made it a satisfying and enjoyable experience.
-  Thomas S.
-  Chief Operating Officer
-
-  Incredible end result! Our sales increased over 400% when we worked with Sunnyside. Highly recommended!
-  Jennie F.
-  Business Owner
-
-  About
-  Services
-  Projects
-  Contact
-
-  <div class="attribution">
-    Challenge by <a href="https://www.frontendmentor.io?ref=challenge" target="_blank">Frontend Mentor</a>. 
-    Coded by <a href="#">Your Name Here</a>.
-  </div>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"> <!-- displays site properly based on user's device -->
+    <link rel="icon" type="image/png" sizes="32x32" href="./images/favicon-32x32.png">
+    <link rel="stylesheet" href="./build/css/app.css">
+    <title>Frontend Mentor | Sunnyside agency landing page</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="header__contenido content">
+        <h1>sunnyside</h1>
+        <div class="mobile">
+          <img src="/build/img/" alt="icon-hamburger">
+        </div>
+        <nav class="header__nav desktop">
+          <a href="#">About</a>
+          <a href="#">Services</a>
+          <a href="#">Projects</a>
+          <a href="#">Contact</a>
+        </nav>
+      </div>
+      <p>We are creatives</p>
+      <img class="arrow" src="/build/img/icon-arrow-down.svg" alt="arrow-image">
+    </header>
+
+    <main class="main">
+      <div class="main__transform ">
+        <section class="main__container content">
+          <h2>Transform your brand</h2>
+          <p>We are a full-service creative agency specializing inhelping brands grow fast. 
+            Engage your clients through compelling visuals that do most of the marketing for you.</p>
+          <a href="">Learn more <span></span></a>
+        </section>
+        <img class="main__image--mobile" src="/build/img/mobile/image-transform.jpg" alt="egg-image-version-phone">
+        <img class="main__image--desktop" src="/build/img/desktop/image-transform.jpg" alt="egg-image-version-desktop">
```

Comment:
remove the dashes in the alt name, the alt name is suppose to be shown when the images couldn't load, so is a text that the user will read, also in terms of accessibility if someone blind uses software to spell your webpage it will get confused with the dashes